domain/todo_usecase_interface.go

```diff
@@ -1,4 +1,9 @@
 package domain
 
+import (
+	"context"
+)
+
 type TodoUseCaseInterface interface {
```

Comment:
Note: It is no need to declare interface name as `TodoUseCaseInterface` you can declare as `TodoUseCase`